week-5/F-webchat/exercise-3.js

```diff
@@ -42,4 +45,8 @@ Task 4
 Use the setInterval function to reload automatically the messages of your webchat every 2 seconds.
 The code responsible to show the messages in the page is in exercise-1.js, so you will need to write your code there :-)
 */
+function displayMessage(){
+    console.log('chats');
```

Comment:
You've already written the code to able to do this. You need to now call that function from `displayMessage`